libraries/core/src/main/java/com/paritytrading/philadelphia/FIXConnection.java

```diff
@@ -100,13 +100,15 @@ public class FIXConnection implements Closeable {
      * @param config the connection configuration
      * @param listener the inbound message listener
      * @param statusListener the inbound status event listener
+     * @param initialMillis the initial time for the connection in epoch time.
+     *   For production normally System.currentTimeMillis()
```

Comment:
Let's align this with [`setCurrentTimeMillis()`](https://github.com/paritytrading/philadelphia/blob/41051aad4679ca3823b342db059e40f7bc582c41/libraries/core/src/main/java/com/paritytrading/philadelphia/FIXConnection.java#L336): `currentTimeMillis`, "the current time in milliseconds"

Reply:
alignment done in [29ebc04](https://github.com/paritytrading/philadelphia/pull/302/commits/29ebc049f7e659daabc7aca7264b2a1bf0820b9d)